app/assets/stylesheets/components/_autocomplete.scss

```diff
@@ -0,0 +1,90 @@
+@import url(asset-path("accessible-autocomplete/dist/accessible-autocomplete.min.css"));
```

Comment:
Where is this coming from? Can't see a node_module or anything being added?

Reply:
I think it must be coming from https://github.com/alphagov/govuk_publishing_components/blob/master/package.json#L16.

We have the govuk_publishing_components gem, which I believe includes all the bundled node_modules. Not quite sure how it all works - Rails magic! It resolves though:

<img width="787" alt="Screen Shot 2019-07-31 at 09 50 42" src="https://user-images.githubusercontent.com/5111927/62197955-b4117080-b378-11e9-8ab5-4519a3b6a3dd.png">

...and breaks if I change the path.